tasks/README.md

```diff
@@ -534,6 +534,10 @@ Name | Summary | Category
 `task610_conllpp_ner` | Recognize and label proper nouns (Named Entity Recognition) | NER/Labeling  
 `task615_moviesqa_answer_generation` | Given a question from an open movie database, generate an answer for that. | Answer Generation
 `task561_spl_translation_en_bg` | Language Translate of Dataset Card for ALT from English language to Bengali language while preserving named entities in the original language | Language Translation
+`task757_msr_sqa_question_generation` | Given a passage from msr_sqa dataset, generate a question based on the information presented | Question generation
+`task758_msr_sqa_question_answer_generation` | Given a passage from msr_sqa dataset and a question based on that passage, generate a correct answer based on the passage | Answer generation
+`task759_msr_sqa_incorrect_answer_generation` | Given a passage from msr_sqa dataset and a question based on that passage, generate an incorrect correct answer based on the passage | Incorrect Answer Generation
+`task760_msr_sqa_long_text_generation` | Given a passage from msr_sqa dataset in tabular format, generate a long text passage based on the information in the tabular data | Long Text generation
```

Comment:
Could you update the description of the tasks here? (mentions of "passage")

Reply:
Sure, will update it.